enhancements/network/on-prem-service-load-balancers.md

```diff
@@ -229,35 +230,123 @@ doing all of your testing in a virtual network built on top of it.
 
 #### Operator
 
-This is the first of two alternatives for how we might integrate MetalLB in
-OpenShift.
-
 We must also create an operator for MetalLB.  We should develop an operator
 that is generally useful to the MetalLB community.  We should also have an
-OpenShift version of this operator for our use.
+OpenShift version of this operator for our use, which would contain any needed
+OpenShift specific customizations.
 
-It is assumed that the MetalLB operator would be managed by OLM as an optional
-additional component to be installed on on-premise clusters.  However, in the
-[ROADMAP.md
+The MetalLB operator will be managed by OLM as an optional additional component
+to be installed on on-premise clusters.  However, in the [ROADMAP.md
 document](https://github.com/openshift/enhancements/blob/master/ROADMAP.md),
 there is an item to "Front the API servers and other master services with
 service load balancers".  If this functionality is required at install time,
-the details on management of this operator may be revisited.
+the details on management of this operator may be revisited, since MetalLB may
```

Comment:
@russellb FYI @aojea has plans to make this not required, at least for Go-based things, by making client-go able to accept multiple API servers and balance/fail-over when required, instead of funneling into a single point of failure like the current keepalived/haproxy metal model or the slow-as-molasses-to-update cloud LB model, both of which we've had tons of problems with.

Reply:
thanks!  that's good to know.

I think the impact of that would mean it's less likely that MetalLB would ever be a required component.